web/impact/impact/tests/test_graphql.py

```diff
@@ -398,27 +396,3 @@ def test_non_staff_user_can_access_finalist_graphql_view(self):
                     }
                 }
             )
-
-    def test_query_expert_by_email(self):
```

Comment:
Why is this test being removed?

Reply:
I removed this because we no longer have the functionality to get query by email on expert profile(graphql)